src/expiry-date/index.jsx

```diff
@@ -24,26 +22,13 @@ const ExpiryDate = ({ date, expiry, dateFormat, unit, showUrgent, showNotice })
     month: differenceInMonths(expiry, now)
   };
 
-  const displayUnit = diff['day'] <= 7 ? 'day' : (diff['day'] <= 28 ? 'week' : 'month');
-  const displayDiff = displayUnit === 'day' ? diff[displayUnit] : diff[displayUnit] + 1;
-  const urgent = displayDiff <= showUrgent;
-
-  let contentKey = 'diff.standard';
-
-  if (isBefore(expiry, now)) {
-    contentKey = 'diff.expired';
-  } else if (urgent) {
-    contentKey = displayDiff === 1 ? 'diff.singular' : 'diff.plural';
-  }
-
   return (
     <Fragment>
-      {format(date, dateFormat)}
-      {diff[unit] <= showNotice && (
-        <span className={classnames('notice', { urgent })}>
-          <Snippet diff={displayDiff} unit={displayUnit}>{contentKey}</Snippet>
-        </span>
-      )}
+      { format(date, dateFormat) }
+      {
+        diff[unit] <= showNotice &&
```

Comment:
I'd be tempted to move this condition to inside the `<Countdown />` component so the logic is entirely encapsulated there.

Reply:
Otherwise we end up calculating all of the diffs in this component and then _again_ inside the `<Countdown />` component.